README.md

```diff
@@ -1,8 +1,29 @@
-# AntAlmanac
-![AntAlmanac Logo](/public/logo.png)
-
-## Get Setup to Develop Locally
-1. Clone the AntAlmanc repository  
+![AntAlmanac](/public/banner.png)
+## About
+AntAlmanac is a schedule planner website for classes at UC Irvine. Here are some of its features:
+- Search bar to easily find classes by department (e.g COMPSCI), course code (e.g. ICS 31), and keywords (e.g. artificial intelligence)
+- Calendar view to make it easier to see class times and spot conflicts
+- Quick links to professor reviews, prerequisites, grade distributions, and past enrollment data.
+- Notification system to tell you when a class goes from full to waitlist
+- See where your classrooms are with the map view
+- Clean UI (see below)
+![project screenshot](https://user-images.githubusercontent.com/48658337/177026240-be0f79b4-e909-486b-aa52-c1a435983781.png)
+### Tech Stack
+Our site is a React single page application bootstrapped with create-react-app and hosted on Github Pages. We use the Material UI component library for most of our styling. The site gets its course data from the [PeterPortal API](https://api.peterportal.org/), which is also maintained by ICSSC.
+### History
+This project started as a side project by two UCI students in 2018 and soon was transferred to the ICS Student Council Projects Committee. Since then, the project has continued to evolve and grow with successive generations of projects committee members. Right now the most tenured members of the AntAlmanac team started in 2021.
```

Comment:
```suggestion
AntAlmanac was created in 2018 by a small group of students under the leadership of @the-rango.  
They formed an AntAlmanac club to recruit other students and work on new features, so that the website would live on even after its makers graduated.  
In 2019, @devsdevsdevs took over as AntAlmanac Project Lead and oversaw a massive rewrite of the codebase, laying the foundation for the AntAlmanac that we know and love today.  
In 2020, AntAlmanac was adopted by the ICSSC Projects Committee. ICSSC continues provides funding, marketing, and engineering to support the growing number of users and open-source developers that make up our AntAlmanac Community.

| Year | Project Lead |
| --- | --- |
| 2018-2019 | @the-rango (founder) |
| 2019-2021 | @devsdevsdevs |
| 2021 - 2022 | @ChaseC99 |
| 2022 - | @EricPedley |
```

Reply:
Looks good